health/QUICKSTART.md

```diff
@@ -138,7 +132,9 @@ killall -USR2 netdata
 
 ## What's next?
 
-To learn about all of Netdata's health configuration options, view the [reference guide](/health/REFERENCE.md).
+To learn about all of Netdata's health configuration options, view the [reference guide](/health/REFERENCE.md) and
+[daemon configuration](/docs/agent/daemon/config/#health-section-options) for additional options available in the
```

Comment:
This directory tree `/docs/agent/daemon/config/` is not present in the current `master` branch, can this create an error?

Reply:
Yep, you're right. That's the pathing once it's on the Learn site, but it will break here. Fixed.

Reply:
I reviewed your fixes now, and I confirmed that you fixed the directory inside the file `health/REFERENCE.md` , but the last two commits did not bring a fix for this file.

Reply:
Got it. Thanks.